COLLABORATOR_GUIDE.md

```diff
@@ -427,7 +428,9 @@ Node.js uses three Deprecation levels:
   deprecated status.
 
 * *End-of-life* refers to APIs that have gone through Runtime Deprecation and
-  are ready to be removed from Node.js entirely.
+  are no longer subject to the semantic versioning and [support policies](LTS)
```

Comment:
As above, the link here doesn't provide clarity. Can you be specific about some aspect of deprecated APIs that is different than everything else other than the semantic version stuff? If not, let's remove `and support policies`.